docs/source/examples/demo_transformer_model_train_save_upload_to_openSearch.ipynb

```diff
@@ -100,51 +115,84 @@
    "metadata": {},
    "source": [
     "## Step 1: Read synthetic queries and train/fine-tune model using a hugging face sentence transformer model\n",
-    "With a synthetic queries zip file, users can fine tune a sentence transformer model. The `train` function will import synthestic queries, load sentence transformer example and train the model using a hugging face sentence transformer model.\n",
+    "With a synthetic queries zip file, users can fine tune a sentence transformer model. \n",
     "\n",
-    "    \"\"\"\n",
-    "    Description:\n",
-    "    read the synthetic queries and use it to fine tune/train a sentence transformer model to save a zip file\n",
+    "The `SentenceTransformerModel` class will inititate an object for training, exporting and configuring the model. `SentenceTransformerModel` class takes the following arguments. \n",
```

Comment:
```suggestion
    "The `SentenceTransformerModel` class will inititate an object for training, exporting and configuring the model. `SentenceTransformerModel` constructor takes the following arguments. \n",
```

Reply:
I think we will show the below doc string on https://opensearch-project.github.io/opensearch-py-ml/reference/sentencetransformer.html# too , right? 
That seems not easy to maintain. How about we just show doc strings on https://opensearch-project.github.io/opensearch-py-ml/reference/sentencetransformer.html#, we add reference here. So we don't need to change two places in future.

Reply:
good idea, I will remove the doc string on the demo notebook and references to the  https://opensearch-project.github.io/opensearch-py-ml/reference/sentencetransformer.html#. It will make the demo notebook in a shorter length and make it more readable.

Reply:
make the edit with hyper link:
example:
The train function will import synthestic queries, load sentence transformer example and train the model using a hugging face sentence transformer model. Plese visit the [SentenceTransformerModel.train](https://opensearch-project.github.io/opensearch-py-ml/reference/api/sentence_transformer.html#opensearch_py_ml.sentence_transformer_model.SentenceTransformerModel.train) for API Reference .